package.json

```diff
@@ -71,7 +71,7 @@
     "server-only": "^0.0.1"
   },
   "peerDependencies": {
-    "@auth/core": "^0.36.0",
+    "@auth/core": "^0.37.0",
```

Comment:
If we land this, we should also update the docs that tell you to install `@auth/core@0.36.0`

Reply:
I'm guessing you mean `0.37.0`.

Reply:
We should update the docs that tell you to install `@auth/core@0.36.0` to tell you to install `@auth/core@0.37.0`

Reply:
Yep! Updated!